conan/api/subapi/cache.py

```diff
@@ -100,16 +101,19 @@ def clean(self, package_list, source=True, build=True, download=True, temp=True,
                         rmdir(folder)
         if backup_sources:
             backup_files = self.conan_api.cache.get_backup_sources(package_list, exclude=False, only_upload=False)
+            ConanOutput().verbose(f"Cleaning {len(backup_files)} backup sources")
             for f in backup_files:
                 remove(f)
 
         for ref, ref_bundle in package_list.refs().items():
+            ConanOutput(ref.repr_notime()).verbose("Cleaning recipe cache contents")
             ref_layout = cache.recipe_layout(ref)
             if source:
                 rmdir(ref_layout.source())
             if download:
                 rmdir(ref_layout.download_export())
             for pref, _ in package_list.prefs(ref, ref_bundle).items():
+                ConanOutput(pref).verbose("Cleaning package cache contents")
```

Comment:
Isn't this ``pref`` too verbose with timestamps, etc?

Reply:
The `str()` of a pref is just `name/version:pkgid` so this is as verbose as the `ref.repr_notime()` from above